src/main/java/uk/gov/hmcts/reform/sscs/service/JudicialRefDataService.java

```diff
@@ -54,15 +55,18 @@ private String getJudicialUserFullName(@NonNull String personalCode, IdamTokens
     }
 
     private static String splitInitials(JudicialUser judicialUser) {
-        String initials = judicialUser.getInitials();
-
-        if (isNotBlank(initials)) {
-            StringBuilder result = new StringBuilder();
-            for (int i = 0; i < initials.length(); i++) {
-                result.append(initials.charAt(i));
-                result.append(" ");
+        String fullName = judicialUser.getFullName();
+        String surname = judicialUser.getSurname();
+        String title = judicialUser.getTitle();
+
+        if (isNotBlank(fullName) && isNotBlank(surname) && isNotBlank(title)) {
+            String givenName = fullName.substring(title.length(), fullName.length() - surname.length()-1).trim();
```

Comment:
might be simpler with split
String givenName = fullName.substring(title.length(), fullName.length() - surname.length()-1).split(" ");
givenName[0].charAt(0) + " " + (givenName.length > 1 ? givenName[1].charAt(0) : "");